src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments.html.twig

```diff
@@ -62,12 +62,12 @@
           <td>{{ payment.amount }}</td>
           <td>{% if payment.invoiceNumber %}{{ payment.invoiceNumber }}{% endif %}</td>
           <td class="text-right">
-            <button class="btn btn-sm btn-outline-secondary">
+            <button class="btn btn-sm btn-outline-secondary js-payment-details-btn">
               {{ 'Details'|trans({}, 'Admin.Global') }}
             </button>
           </td>
         </tr>
-        <tr>
+        <tr class="d-none">
           <td colspan="6">
             <p class="mb-0">
               <strong>{{ 'Card number'|trans({}, 'Admin.Global') }}</strong>
```

Comment:
Is this information really displayed in the back office? Is it the whole card number ou just a few numbers with the rest hidden?

Reply:
I guess it could be both, it depends on what merchant/module sets for this value.

Reply:
It is a bit annoying not to be able to know because it is a very bad practice regarding security, even if it is a setting enabled by the merchant himself or the payment module he is using... we should not allow the display of this information, or at least make sure it hides a part of it. I got in touch with the modules teams to know more about it, I keep you posted!

Reply:
It seems that our major payment modules do not even use this bloc, this information is not retrieved. And it is better because we absolutely need to prevent the back office from retrieving it. I will try various modules to be sure that this bloc is deprecated and ask for its deletion if it is the case.

Reply:
I don't think we can delete it, we had client that uses this Payments block.

Reply:
Okay, it's interesting! Can you send me an email to let me know what payment module(s) are they using?

Reply:
It was a customized solution, but this feature was needed.

Reply:
So, to keep you posted, it has been decided to delete this details line. @colinegin has suggested adding a hook at this place, I'm okay if it can be useful, I just need to check the impact before.

Reply:
Can we first merge this one and perform behavior modifications in dedicated PR? It would be way better imo.

Reply:
If you prefer to do so, it suits me fine. :-)

Reply:
Issue created here https://github.com/PrestaShop/PrestaShop/issues/15893 for this topic